compiler/rustc_const_eval/src/interpret/eval_context.rs

```diff
@@ -631,14 +631,10 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 // the last field).  Can't have foreign types here, how would we
                 // adjust alignment and size for them?
                 let field = layout.field(self, layout.fields.count() - 1);
-                let (unsized_size, unsized_align) =
-                    match self.size_and_align_of(metadata, &field)? {
-                        Some(size_and_align) => size_and_align,
-                        None => {
-                            // A field with an extern type. We don't know the actual dynamic size
-                            // or the alignment.
-                            return Ok(None);
-                        }
+                let Some((unsized_size, unsized_align)) = self.size_and_align_of(metadata, &field)? else {
+                        // A field with an extern type. We don't know the actual dynamic size
```

Comment:
Indentation seems off here?

Reply:
Thanks for pointing that out. Rustfmt currently doesn't support `let else`. I aligned formatting manually but missed that spot I guess.